### PR TITLE
Fixed subdomain installations

### PR DESCRIPTION
**Pull request by fourteentrees**
I've split the config into two different files: one with __PATH__ and one without. The one without is copied if the path is equal to /, the one with otherwise.

Tested with YNH 11.2.9.1 stable on Google Cloud. Had to mess with scripts a bit because my brain decided to just...stop working and forget how to do Bash comparisons.

Easy fix, it works, either way you get the XBackBone login screen after installation.

### CODE REVIEW

**Comment by haydenwalker980** [comment]
!testme

**Comment by fourteentrees** [comment]
Just realised conflicts. Oops.

Shouldn't be too hard to integrate my changes, though.

**Comment by ericgaspar** [comment]
!testme

File: conf/config-subdomain.php [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+    'ldap' => array(
+        'enabled' => true,
+        'schema' => 'ldap',
+        'host' => 'ldap://127.0.0.1',
+        'port' => 389,
+        'base_domain' => 'dc=yunohost,dc=org',
+        'search_filter' => '(&(|(objectclass=posixAccount))(uid={{username}})(permission=cn=__APP__.main,ou=permission,dc=yunohost,dc=org))',
+        'user_domain' => 'ou=Users',
+        'rdn_attribute' => 'uid=',
```
Comment (ericgaspar) [inline conf/config-subdomain.php:25]: these lines differ from the 'config-withpath.php' file

Comment (ericgaspar) [reply conf/config-subdomain.php:25]: also, the change_url will need to be adapted accordingly. It was certainly not working thought

Comment (fourteentrees) [reply conf/config-subdomain.php:25]: that's weird, one minute

**Comment by ericgaspar** [comment]
!testme

**Review by ericgaspar — APPROVED** [review]

**Pull request merged**